spring-cloud-dataflow-shell/src/main/java/org/springframework/cloud/dataflow/shell/command/FieldValueCounterCommands.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.shell.command;
+
+import java.text.NumberFormat;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.dataflow.rest.client.FieldValueCounterOperations;
+import org.springframework.cloud.dataflow.rest.resource.FieldValueCounterResource;
+import org.springframework.cloud.dataflow.rest.resource.MetricResource;
+import org.springframework.cloud.dataflow.shell.config.DataFlowShell;
+import org.springframework.cloud.dataflow.shell.converter.NumberFormatConverter;
+import org.springframework.hateoas.PagedResources;
+import org.springframework.shell.core.CommandMarker;
+import org.springframework.shell.core.annotation.CliAvailabilityIndicator;
+import org.springframework.shell.core.annotation.CliCommand;
+import org.springframework.shell.core.annotation.CliOption;
+import org.springframework.shell.table.BeanListTableModel;
+import org.springframework.shell.table.CellMatchers;
+import org.springframework.shell.table.Formatter;
+import org.springframework.shell.table.SimpleHorizontalAligner;
+import org.springframework.shell.table.Table;
+import org.springframework.shell.table.TableBuilder;
+import org.springframework.shell.table.TableModel;
+import org.springframework.stereotype.Component;
+
+/**
+ * Commands for interacting with Field Value Counter analytics.
+ * 
+ * @author Eric Bottard
+ */
+@Component
+public class FieldValueCounterCommands extends AbstractMetricsCommands implements CommandMarker {
+
+	protected FieldValueCounterCommands() {
+		super("Field Value Counter");
+	}
+
+	private static final String DISPLAY_COUNTER = "field-value-counter display";
+
+	private static final String LIST_COUNTERS = "field-value-counter list";
+
+	private static final String DELETE_COUNTER = "field-value-counter reset";
```

Comment:
If we decide to go with `reset` operation, I think we can rename this operation to `reset`